rules/manager.go

```diff
@@ -175,6 +177,7 @@ type Group struct {
 	opts                 *ManagerOptions
 	evaluationDuration   time.Duration
 	mtx                  sync.Mutex
+	evaluationTimestamp  time.Time
```

Comment:
This field should be grouped with `evaluationDuration`, probably under `mtx` (similar mutex clarity problem as above).